charter/index.html

```diff
@@ -1,303 +1,245 @@
-                interoperable
-                implementations</a> of every feature defined in the specification, where
-                interoperability can be verified by passing open test suites, and two or
-                more implementations interoperating with each other.</p>
-
-                <p>There should be testing plans for each specification, starting from the earliest drafts.</p>
-
-                <p>To promote interoperability, the group will publish <a href="https://solidproject.org/ED/qa">Solid QA</a> detailing testing policy, processes, and procedures.</p>
-
-                <!-- Horizontal review -->
-                <p>Each specification should contain sections detailing all known security and privacy implications for implementers, Web authors, and end users.</p>
-
-                <p>Each specification should contain a section on accessibility that describes the benefits and impacts, including ways specification features can be used to address them, and
-                recommendations for maximising accessibility in implementations.</p>
-
-                <!-- Design principles -->
-                <p>This Working Group expects to follow the TAG <a href="https://www.w3.org/TR/design-principles/">Web Platform Design Principles</a>.</p>
-            </div>
+          <ul class="out-of-scope">
+            <li>
+                      Definition of identity mechanisms such as WebID and DID
+            </li>
+            <li>
+                      Definition of linked data formats
+            </li>
+          </ul>
         </section>
 
-        <section id="deliverables">
-            <h2>
-                Deliverables
-            </h2>
-            <p>
-                More detailed milestones and updated publication schedules are available on the
-                <a class=todo href="">group publication status page (LINK TBD)</a>.
-            </p>
-
-            <div id="normative">
-                <h3>
-                    Normative Specifications
-                </h3>
-                <p>
-                    The Solid Working Group will deliver the following W3C normative specifications:
-                </p>
+      </section>
+
+      <section id="deliverables">
+        <h2>
+          Deliverables
+        </h2>
+
+        <p>Updated document status is available on the <a class="todo" href="">group publication status page (LINK TBD)</a>.</p>
+
+        <section id="normative">
+          <h3>
+            Normative Specifications
+          </h3>
+          <p>
+            The Working Group will deliver the following W3C normative specifications:
+          </p>
                 <dl>
-                    <dt id="protocol" class="spec">Solid Protocol v1.0</dt>
+                    <dt id="solid-protocol" class="spec"><a href="https://solidproject.org/TR/protocol">Solid Protocol v1.0</a></dt>
                     <dd>
```

Comment:
The Solid TR document is not the deliverable of the WG. This link is misleding. I replaced it with an explicit reference, for the deliverable, to the current specification (but seen as a separate document).
```suggestion
                    <dt id="solid-protocol" class="spec">Solid Protocol v1.0</dt>
                    <dd>
                    <p><strong>Reference:</strong> <a href="https://solidproject.org/TR/protocol">Solid Protocol v.0.10.0</a>, eds. Sarven Capadisli, Tim Berners-Lee, Ruben Verborgh, Kjetil Kjernsmo. Solid Technical Report 2023.</p>
```